requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy==1.20.3
+scikit-learn==0.24.2
+matplotlib==3.4.2
+pandas==1.2.4
```

Comment:
Do you think we should pin the versions to exactly these versions, or maybe use >= ? It worked with previous versions, and I don't think it's likely that the code we use will break in the future...

Reply:
I am keeping the exact version in order to keep it fine, If a breaking change is released. We could look into this later I guess.